diff --git a/src/sunstone/public/build.sh b/src/sunstone/public/build.sh
index 58f7ff94e..8f3b7d0a9 100755
--- a/src/sunstone/public/build.sh
+++ b/src/sunstone/public/build.sh
@@ -26,13 +26,6 @@ dependencies() {
 }
 
 install_patch() {
-
-    npm install
-
-    bower install --force --allow-root --config.interactive=false
-
-    (cd bower_components/no-vnc/ && npm install && ./utils/use_require.js --clean --as amd && sed -i -e "s/'\.\//'\.\.\/bower_components\/no-vnc\/lib\//g" lib/rfb.js )
-
     PATCH_DIR="./patches/"
 
     for i in `ls ${PATCH_DIR}` ; do
