diff --git a/SConstruct b/SConstruct
index e758fc2dd..c70b7254d 100755
--- a/SConstruct
+++ b/SConstruct
@@ -182,6 +182,9 @@ main_env.Append(sunstone=ARGUMENTS.get('sunstone', 'no'))
 # Docker-machine addon generation
 main_env.Append(docker_machine=ARGUMENTS.get('docker_machine', 'no'))
 
+# Man pages files generation
+main_env.Append(manpages=ARGUMENTS.get('manpages', 'no'))
+
 if not main_env.GetOption('clean'):
     try:
         if mysql == 'yes':
@@ -277,6 +280,7 @@ build_scripts=[
     'src/ipamm/SConstruct',
     'src/sunstone/public/locale/languages/SConstruct',
     'src/sunstone/public/SConstruct',
+    'share/man/SConstruct',
     'share/rubygems/SConstruct',
     'src/im_mad/collectd/SConstruct',
     'src/client/SConstruct',
